environment.d.ts

```diff
@@ -0,0 +1,24 @@
+declare global {
+  namespace NodeJS {
+    interface ProcessEnv {
+      EXPO_ENV: string;
```

Comment:
Zod will be helpful here too to prevent needing to declare a type and then check at runtime